map/framework.cpp

```diff
@@ -1326,16 +1328,15 @@ void Framework::SetCurrentCountryChangedListener(TCurrentCountryChanged const &
   m_lastReportedCountry = kInvalidCountryId;
 }
 
-void Framework::PokeSearchInViewport()
+void Framework::PokeSearchInViewport(bool forceSearch)
 {
   if (!m_isViewportInitialized || !IsViewportSearchActive())
     return;
 
-  auto & intent = m_searchIntents[static_cast<size_t>(search::Mode::Viewport)];
-  auto & params = intent.m_params;
+  auto params = m_searchIntents[static_cast<size_t>(search::Mode::Viewport)].m_params;
```

Comment:
I guess we need the comment about the intentional copy back.

Reply:
Done